src/OrchardCore.Modules/OrchardCore.Users/Views/RegistrationSettings.Edit.cshtml

```diff
@@ -90,7 +90,7 @@
         <input type="checkbox" class="custom-control-input" asp-for="NoUsernameForExternalUsers" data-external-registration />
         <span asp-validation-for="NoUsernameForExternalUsers"></span>
         <label class="custom-control-label" asp-for="NoUsernameForExternalUsers">@T["Do not ask username"]</label>
-        <span class="hint">— @T["When a new user logs in with an external provider, he is not required to provide a local username.You can customize how it looks by providing an IExternalLoginEventHandler or write a script."]</span>
+        <span class="hint">— @T["When a new user logs in with an external provider, he is not required to provide a local username. You can customize how it looks by providing an IExternalLoginEventHandler or write a script."]</span>
```

Comment:
```suggestion
        <span class="hint">— @T["When a new user logs in with an external provider, they are not required to provide a local username. You can customize how it looks by providing an IExternalLoginEventHandler or writing a script."]</span>
```

Reply:
Do you think this is `You can customize how it looks by` or `You can customize how it works` ? @Piedone

Reply:
Good catch and yes, it indeed should rather reference "works". I've changed it.